webui/react/src/hooks/useMetricNames.ts

```diff
@@ -16,12 +16,8 @@ const useMetricNames = (
   quickPoll?: boolean,
 ): Loadable<Metric[]> => {
   const [metrics, setMetrics] = useState<Loadable<Metric[]>>(NotLoaded);
-  const [actualExpIds, setActualExpIds] = useState<number[]>([]);
+  const actualExpIds = useMemo(() => experimentIds, [experimentIds]);
```

Comment:
wrap `experimentIds` with `useMemo` since `experimentIds` is a raw object (array)

Reply:
I'm not sure about using `useMemo` like this.. if the reference of `experimentIds` changes (without value change), wouldn't `actualExpIds` change too?

Reply:
this line doesnt mean much other than making `experimentIds` a state variable. 
i think its ok to change `actualExpIds` when `experimentIds` is changed, right? especially its cheap to compute

Reply:
changed my mind. i revert the change. this is not fault tolerant but i think its cleaner, and this is what review is for to catch the future bug